src/managers/GuildMemberManager.js

```diff
@@ -127,6 +127,23 @@ class GuildMemberManager extends BaseManager {
     return this._fetchMany(options);
   }
 
+  /**
+   * Search for members in the guild based on a query
```

Comment:
```suggestion
   * Search for members in the guild based on a query.
```
i believe a full stop should be used here (to be consistent with the descriptions for other methods)